src/main/lrsql/admin/protocol.clj

```diff
@@ -29,3 +29,6 @@
 (defprotocol AdminStatusProvider
   (-get-status [this params]
     "Get various LRS metrics."))
+
+(defprotocol ActorDeleter
```

Comment:
Same name/group change as before I'd do this more somehting like `AdminStatementOperationProvider`

Reply:
Chiming in: I actually prefer the general approach of Daniel's name since it's more concise. I do agree it should be more generic like `ActorManager` however.

Reply:
I actually wonder if it can be combined with `AdminStatusProvider` since both deal with the LRS data that the admin has scope over.

Reply:
i dunno. im envisioning a statement operations featureset (i know, its ugly but im sure there will be more), and status is literally a health check with some metrics.

Reply:
So maybe `StatementManager` or `DataManager`? I just dislike the idea of naming the protocol an "operations provider" since it's very easy to confuse with the idea of providing data from the DB (i.e. what `AdminStatusProvider` does)

Reply:
OTOH it could be useful to prefix the protocol name with "admin," as you suggested, since that would distinguish it from statement/actor ops that anyone can use (i.e. the xAPI spec).

Reply:
That was my thinking exactly. It deliniates from normal statement ops and specifies admin statement ops. you can maybe combine the metrics one, thats fine, but i want the idea that its admin AND that its statement operations in there. if you dont like the word operations im open to alts.

Reply:
Here's another option: `AdminResource`. Since the protocols defined by the lrs lib are `StatementsResource`, `DocumentResource`, and the like, so going with that scheme highlights that these are (admin) actions on the basic LRS data.

Reply:
A couple more names:
- `AdminLRSManager`: Similar to the names of the other "admin manager" protocols.
- `AdminResource`: Goes with the precedent set by the lrs protocols like `StatementsResource` and `DocumentsResource`.

We'll let Daniel come up or select a name that he likes.

Reply:
@kelvinqian00 `AdminLRSManager` nails it. Love it. @invaliduser you good w/that since its your feature?

Reply:
went with AdminLRSManager